pkgs/development/tools/glamoroustoolkit/GlamorousToolkit-GetImage

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash --packages curl wget
+#
+# Download the latest GlamorousToolkit image to the current directory, which must be empty
+#
+set -e
+
+if [ ! -z "$(ls)" ]
+then
+	echo "Warning: the directory is not empty, continuing may overwrite some files"
+	read -p "Continue? (Y/N): " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || exit 1
+fi
+
+GTVERSION=$(curl -s https://api.github.com/repos/feenkcom/gtoolkit/releases/latest 2>/dev/null | grep tag_name | cut -d '"' -f 4)
```

Comment:
Is it OK to use a version that not matches the installed version?

Reply:
You mean that the VM version doesn't match the image version?

GT is built on smalltalk, typically the VM changes multiple times a year while the image changes multiple times per day.

Reply:
Nice!

Reply:
Thanks. :-)

The summary at the bottom of the page (GitHub conversation page) is saying that you are requesting a change here:

![2023-08-06_22-39](https://github.com/NixOS/nixpkgs/assets/2062166/170383a5-b90f-486c-933a-5db10b675144)

However the conversation is showing as resolved, and there's nothing to commit, so I'm not sure what's required.

Thanks again for all your help.

Reply:
GitHub requires a final review in order to remove these.